packages/next-steps/types.ts

```diff
@@ -39,12 +43,26 @@ export interface NextStepsProps {
   /**
    * Optional ctas displayed below the heading and description.
    */
-  ctas?: ActionsProps['ctas']
+  cta?: {
+    copy: string
+    href: string
+    ctaText: string
+  }
```

Comment:
So we are keeping the cta option but limiting to one?

Reply:
Oops! `tertiaryCta` was a placeholder when I wasn't sure if we were still enabling CTAs in the `Intro` component.

I've removed references to `tertiaryCta` and updates both the component's types and props files.